GameDesign/Assets/Scripts/Managers/ClickManager.cs

```diff
@@ -97,6 +102,7 @@ public void onCorrectlyClickInvoke()
     }
     public void onMissClickInvoke()
     {
+        audioSource.PlayOneShot(miss, 0.7f);
```

Comment:
We need to remove this, otherwise it will also sound when we click a masked person, overlaying with the already existing sound